requirements/travis.in

```diff
@@ -0,0 +1,5 @@
+# Basic requirements for Travis CI
+
+-c constraints.txt
+
+codecov
```

Comment:
Ah, I didn't notice in my previous comment that you'd created `tox.in` instead of `travis.in` (we use the latter in most of our repos).  It's not urgent to fix, but we generally just have one of the two, containing codecov, tox, and tox-battery.